deno/base/1.37-dev/Dockerfile

```diff
@@ -34,14 +34,16 @@ RUN /usr/bin/apt-install Aptfile
 
 USER noteable
 
-# use the deno installer and upgrade to the latest (canary) build
+# use the deno installer and upgrade to the specified (canary) build
 # https://docs.deno.com/runtime/manual/references/contributing/release_schedule
 ENV DENO_INSTALL=/srv/noteable
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN curl -fsSL https://deno.land/x/install/install.sh | sh
 # deno installs to `DENO_INSTALL/bin`, so we'll add to PATH for follow-on deno commands
 ENV PATH="${DENO_INSTALL}/bin/:${PATH}"
-RUN deno upgrade --canary
+RUN deno upgrade \
+    --canary \
+    --version=87ddd1f04d2b71aadb3a8a5fb3e4ac15988947c1
```

Comment:
Maybe make this an dockerfile arg? This would allow rebuilding without needing to modify the dockerfile

Reply:
I haven't gone down that path with CI builds - where would we pass the new build arg value for future commits? Or would that be in the action config?

Reply:
It would help mostly for local work trying out new versions, but not a blocker